sdk/mixedreality/Azure.MixedReality.RemoteRendering/api/Azure.MixedReality.RemoteRendering.netstandard2.0.cs

```diff
@@ -0,0 +1,198 @@
+namespace Azure.MixedReality.RemoteRendering
```

Comment:
Why is this file modified?

Reply:
I think it was generated after running the api generation script, I will remove it.

Reply:
removed

Reply:
I see that you reverted the changes in a separate commit. It is better to use `git reset HEAD~` till the commit that was making the change, undo the change there and then commit without it. So it does not cause future conflicts for the MixedReality team. Essentially we want to make sure this file is not touched by our branch

Reply:
maybe easier to just get the latest from master, and paste the Chat folder with your changes on top in a new branch, so we avoid any merging issues @glorialimicrosoft ?

Reply:
@juancamilor, that might work expect that Reza also had a commit in the preview4 branch that is outside the Chat folder. I will see what I can do to get rid of the merging issues.

Reply:
The problem seems to have originated from the master branched being merged to your branch early one. You should avoid merging base branch to a child branch and instead rebase your branch onto master.

Reply:
Ya... I think that was the problem that caused the merging issues. Thanks, Reza, I will take out that merge and do rebase instead.

Reply:
Actually, I think that would require some force pushes... I will do it properly in a separate branch then create a brand new PR